src/pages/wallet/WalletStatementPage.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import lodashGet from 'lodash/get';
+import {withOnyx} from 'react-native-onyx';
+import moment from 'moment';
+import Str from 'expensify-common/lib/str';
+import Navigation from '../../libs/Navigation/Navigation';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import ONYXKEYS from '../../ONYXKEYS';
+import compose from '../../libs/compose';
+import CONFIG from '../../CONFIG';
+import WalletStatementModal from '../../components/WalletStatementModal';
+
+const propTypes = {
+    /** The route object passed to this page from the navigator */
+    route: PropTypes.shape({
+
+        /** Each parameter passed via the URL */
+        params: PropTypes.shape({
+
+            /** The statement year */
+            year: PropTypes.string.isRequired,
+
+            /** The statement month */
+            month: PropTypes.string.isRequired,
+        }).isRequired,
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const WalletStatementPage = (props) => {
+    moment.locale(lodashGet(props, 'preferredLocale', 'en'));
```

Comment:
What is this for? Since we're rendering this in OldDot we don't have localization for the statement at this time

Reply:
This is for the modal title, e.g. `December 2021 statement`

Reply:
oh interesting, didn't know moment could do that

Reply:
pretty cool, so if the user's `preferredLocale` is `es` for Spanish then the copy will be `Diciembre 2021`